components/annot-tab.vue

```diff
@@ -44,129 +31,15 @@
     export default {
 
         props: {
-
-            details: { type: Object, required: true },
-            title: { type: String, required: true }
-        },
-
-        inject: [
-
-            "columnToCategoryMap",
-            "dataTable"
-        ],
-
-        data() {
-
-            return {
-
-                // Category of the current annotation tab
-                category: "",
-
-                // String name of the column assigned the 'Subject ID' category in the data table
-                idField: "",
-
-                // Disabled state of the save annotation button
-                saveButtonDisabled: true
-            };
+            activeCategory: { type: String, required: true }
         },
 
         computed: {
-
             ...mapGetters([
-
-                "getColumnsOfCategory"
-            ]),
-
-            filteredDataTable() {
```

Comment:
@surchs Can you briefly account for where these computed property values are now stored in the refactor? In the store? Specialized components? Etc. (`filteredDataTable`, `relevantColumns`, `uniqueValues`, and `uniqueValuesToSubjectMap`).

Reply:
I think the general answer is that the child components are getting their needed data directly from the store now (via getters or direct state access), so there is no more need for these prop chains. 

I think the `filteredDataTable` is now built from the `getUniqueValues` getter draft (not implemented yet) in the `annot-categorical` and from the `getPreviewValues` getter draft (not implemented) in the `annot-continuous-values` component. `relevantColumns` is now coming direct from the store via the `getMappedColumns` draft for the `annot-columns` method. `uniqueValues` is also coming direct from the store. 

Might have missed a few. But the overall idea is: nothing except the `activeCategory` prop has to be passed down to the child components, because with just that information they will go to the store and get themselves their appropriate information